ios/brave-ios/Tests/ClientTests/Resources/html/index.html

```diff
@@ -105,5 +105,23 @@ <h1>Test Procedural Actions</h1>
         </div>
       </div>
     </div>
+    
+    <h1>Test Local Frames</h1>
+    <script>
+    const onload = () => {
```

Comment:
Hmm, do we have to use JS here or can you add the frame as an HTML element?
Just curious if `onload` and JS is necessary.

Reply:
This is just a simple page for unit testing, I think as JS or HTML element is fine.